### ISSUE TITLE
drush_hook_post_sql_sanitize is not fired since 8.1.9

### ISSUE THREAD

**Issue by rsanzante** [drush-ops/drush#2830]
The hook drush_hook_post_sql_sanitize is not fired since drush 8.1.9, it was working with 8.1.8. This hook comes from the drush_hook_post_COMMAND that's documented here: http://api.drush.org/api/drush/drush.api.php/function/drush_hook_post_COMMAND/8.0.x

**Steps to reproduce**

Setup a Drupal 7.x installation. Add a sanitize_extra.drush.inc file in sites/all/drush with this content:

```
<?php 
/**
 * Implements drush_hook_post_COMMAND().
 */
function drush_sanitize_extra_post_sql_sanitize() {
  drush_print_r('drush_sanitize_extra_post_sql_sanitize has been called!');
}
```

Execute sql-sanitize command:
`drush sql-sanitize`

**Expected behavior**

drush_sanitize_extra_post_sql_sanitize hook is called and message is printed.

**Actual behavior**

hook is not called. not message is printed.

** Notes **
Using drush 8.1.8 hook is called. Probably the problem comes from the fact that the sql sanitize command was moved from sql.drush.inc file to SanitizeCommands.php in 8.1.9.

May be other hooks are affected as well.

**Comment by weitzman** [comment]
Right. See Drush9. The sanitize plugin system is better documented there.

**Comment by rsanzante** [comment]
Have this been closed because is fixed or because is not a bug?

**Comment by greg-1-anderson** [comment]
Seems sanitize behavior should have been maintained on the 8.x branch.

**Comment by weitzman** [comment]
OK, leaving this open for Drush 8. Interested parties are welcome to work on it.
